app/src/main/java/org/fossasia/openevent/general/search/SearchViewModel.kt

```diff
@@ -15,19 +15,26 @@ class SearchViewModel(private val eventService: EventService, private val prefer
 
     private val compositeDisposable = CompositeDisposable()
     private val tokenKey = "LOCATION"
+    private val tokenKeyDate = "DATE"
+    private val tokenKeyNextDate = "NEXT_DATE"
 
     val progress = MutableLiveData<Boolean>()
     val events = MutableLiveData<List<Event>>()
     val error = MutableLiveData<String>()
     var searchEvent: String? = null
     val savedLocation by lazy { preference.getString(tokenKey) }
+    val savedDate by lazy { preference.getString(tokenKeyDate) }
+    val savedNextDate by lazy { preference.getString(tokenKeyNextDate) }
 
-    fun loadEvents(location: String) {
+
+    fun loadEvents(location: String,time:String) {
         preference.putString(tokenKey, location)
         val query: String = if (TextUtils.isEmpty(location))
             "[{\"name\":\"name\",\"op\":\"ilike\",\"val\":\"%$searchEvent%\"}]"
-        else
+        else if (time=="Anytime")
```

Comment:
space between time and ==

Reply:
Guys these kind of mistakes shouldn't happen very often. Everytime you push a commit, just check it once using the github files changed tab. I know sometimes they still may happen but frequency will be extremely low and hence reviewers can focus on the logic rather than pointing out formatting mistakes. In the Orga app, reviewers rarely have to point out these mistakes and hence development process is smoother and faster.